### DISCUSSION TITLE
Custom Module Persistence

### DISCUSSION THREAD

**Discussion by bvirgilioamnh** [dfir-iris/iris-web#590] [Q&A]
Hey All,

We're running IRIS via docker-compose, we working on a number of custom modules at the moment and realized that the current method of installation proposed via the docs (using the buildnpush2iris.sh script is not persistent when the container(s) are restarted.

Whats the "best" method to get these custom modules to be persistent within the container(s)?

Thanks!!

**Comment by whikernel** [comment]
Hi @bvirgilioamnh  

Unfortunately for now there's no way to do it simply. We need to think of a new way to install modules. 
Most probably it will be a docker Volume, and then registration on the DB. When the app boots, it will look at registered modules and installed them from the docker Volume. 
Unfortunately I can't give any ETA on this :/
